crates/typos-cli/tests/cmd/stdin-paths0.in/_typos.toml

```diff
@@ -0,0 +1,8 @@
+[files]
```

Comment:
Whats the role of these files?

Reply:
They're an important leftover I forgot about. :D